scripts/us_census/pep/population_estimate_by_race/download.py

```diff
@@ -105,7 +105,7 @@ def _save_data(_url: str, download_local_path: str) -> None:
                 df = pd.read_csv(_url, on_bad_lines='skip', names=cols)
```

Comment:
in line 82 above, pls add context when printing:
https://github.com/datacommonsorg/data/pull/1105/files#diff-cdbbb40fb98f2f2fcc3c38625b8d29b98cc6eac339b4f3d6a3e73e78e9241b37R82

```
print(f'Downloading {file_name} from {_url}')
```

Reply:
we are no longer using this download.py file should I delete this file ?

Reply:
yes lets delete this

Reply:
Deleted